tests/test_cubicBox.py

```diff
@@ -0,0 +1,7 @@
+import mbuild as mb
+import numpy as np
+
+
+def test_box():
```

Comment:
This test checks the functionality of the `mBuild` box class, rather than the functionality of `mBuild_ONA` so we can probably remove this.